### DISCUSSION TITLE
Italian translations

### DISCUSSION THREAD

**Discussion by Luca3082** [microsoft-search/pnp-modern-search#3510] [Q&A]
Hi, is it possible to have to italian translation, especially for the filtering part of the solution?
Or what can I do to contribute?
Thanks in advance

**Comment by kasperbolarsen** [comment]
Hi Luca
Absolutly, and we would welcome an italian version very much :-)
For each supported language we have a js file containing all of the translations for that language:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/5eb91d47-41db-43f2-8167-37b673bdf4c6)

Would you be able to make a PR as per https://microsoft-search.github.io/pnp-modern-search/how-to-contribute/ containing the italian js file
